_scripts/toolbox-create

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/bash -e
 
 echo "## Creating website container..."
-toolbox create -c cockpit-website
+toolbox create -c cockpit-website -r 32
```

Comment:
Can you please give a rationale for this in the commit message, and a comment? I.e. why does the current version not work?

Reply:
The F32 version was when F33 was beta and not working due to a bug. (This was in September.) The idea is that we'd have a consistent repo used.

We can bump it to F33 now. Or just drop it.

Reply:
BTW: I'm not sure why this old commit is showing up on this PR. Weird.

Reply:
Also, the reason why I specified the version is that the default toolbox version you'll have is depending on the version of Fedora you're running. If your host is Fedora 32, you'll have a Fedora 32 toolbox. If the host is Fedora 33, then you'll have a Fedora 33 toolbox. This was to make it consistent for everyone, regardless of the version of Fedora they are running. But as Fedora 33 is now stable and works, we can definitely bump up the version.

Reply:
I thought that the contents of the toolbox isn't very sensitive to the underlying Fedora version, so hardcoding any version will just quickly get out of date and thus we should start with the default version until we run into trouble (at which point we can still pin it). But that's not a strong opinion, if you feel like updating it every now and then, 32 is ok as well.